tests/acceptance/features/apiGraph/createGroup.feature

```diff
@@ -30,13 +30,44 @@ Feature: create group
     And group "mygroup" should exist
 
 
-  Scenario: normal user tries to create a group
+  Scenario Outline: normal user tries to create a group
     Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
     When user "Brian" tries to create a group "mygroup" using the Graph API
     Then the HTTP status code should be "401"
     And group "mygroup" should not exist
+    Examples:
+      | userRole    |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
 
+  Scenario Outline: normal user tries to create a group that already exists
```

Comment:
Won't the above scenario cover this case? Normal users cannot create any group so trying to create an existing group seems to be similar to me.

Reply:
I don't know but trying to create an existing group may be different. Who knows normal users gets permission to create already existing group as a bug.